AEPServices/Sources/ui/AEPUIService.swift

```diff
@@ -14,19 +14,19 @@ import Foundation
 import UIKit
 
 class AEPUIService: UIService {
-    func getFrame() -> CGRect? {
-        var newFrame = CGRect.zero
-        // x is always 0
-        newFrame.origin.x = 0
-        // for fullscreen, width and height are both full screen
-        let keyWindow = UIApplication.shared.getKeyWindow()
-        guard let screenBounds: CGSize = keyWindow?.frame.size else { return nil }
-        newFrame.size = screenBounds
+    func createFullscreenMessage(payload: String, listener: FullscreenMessageDelegate?, isLocalImageUsed: Bool? = nil) -> FullscreenPresentable {
```

Comment:
it is weird to have isLocalImageUsed as `Bool?`, we can use false as the default value